bpf/lib/common.h

```diff
@@ -451,6 +453,7 @@ enum {
 	CB_SRC_LABEL,
 #define	CB_SVC_PORT		CB_SRC_LABEL	/* Alias, non-overlapping */
 	CB_IFINDEX,
+#define	CB_FROM_HOST		CB_IFINDEX	/* Alias, non-overlapping */
```

Comment:
I didn't further review the PR, but while rebasing some of my work to see how it interacts, I noticed that I will need this equivalent bit as an alias for a higher enum as it would conflict with `CB_IFINDEX` in the case where I was going to use it. For context I'm looking at using it to transfer this information acrosse the the tail calls from `ipv[46]_local_delivery()`.

No need to make changes here but I will likely move to shift this CB enum to alias `CB_POLICY` instead (unless you otherwise have reservations on that, then I'll keep searching the higher bits for it, or define a semantically similar bit with a different name and enum.)

Reply:
I had forgotten that I actually removed `CB_FROM_HOST` in a subsequent commit toward the end of the series to address complexity issues. Instead, I've broken `CILIUM_CALL_IPV{4,6}_FROM_LXC` into `CILIUM_CALL_IPV{4,6}_{FROM_LXC,FROM_HOST}`. Then, in the tail called programs, `from_host` is derived from the above entry point. That makes it known at compile time and allows the compiler to reduce the program size to <4096 once policy enforcement is introduced.

I've merged those two commits (second commit) so that `CB_FROM_HOST` doesn't appear in the PR anymore.